### ISSUE TITLE
File version: 0.0.0.0

### ISSUE THREAD

**Issue by Taritsyn** [sebastienros/jint#417]
During installation of the Jint version 2.11.23,  to `csproj`-file are added a reference of the following type:

```xml
<Reference Include="Jint, Version=0.0.0.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f, processorArchitecture=MSIL">
  <HintPath>..\..\packages\Jint.2.11.23\lib\net40\Jint.dll</HintPath>
  <Private>True</Private>
</Reference>
```

`Jint.dll` file has the following properties:

![Jint.dll properties](https://i.imgur.com/oOIGXbm.png)

**Comment by robertmuehsig** [comment]
Ah - we use jint in our desktop application, which is installed via Wix. The first time we used jint it was version 2.10.2 and the fileversion was correct:

![image](https://user-images.githubusercontent.com/756703/31215830-6b88d0ba-a9b1-11e7-9ffc-cbd278fec028.png)

Then I updated the nuget package to 2.11.23 and now the file version is 0.0.0.0, which will lead to a Wix install issue (and I can imaging some other problems as well). 
In case of Wix: Because the new File Version is lower, wix refuses to update the file, so after an update the file version is still 2.10.2.0, but the reference is set to 0.0.0.0.

The 3.0 version doesn't have this problem and has valid file version number, but it seems to be a major change and I want to wait till a stable version 3 is out.

Conclusion: It would be cool if someone could just release a new 2.11.24 version with a fixed file version number.
